src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -213,6 +213,29 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="DotNetCliToolReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="DotNetCliToolReference" _locComment="" -->A project-specific tool that can be ran from the command line. 
+      </xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:complexContent>
+        <xs:extension base="msb:SimpleItemType">
+          <xs:sequence>
+            <xs:element name="Version" />
+          </xs:sequence>
+          <xs:attribute name="Include" type="xs:string">
```

Comment:
Do we need add Include? If so, then we should add it for PackageReference as well - dont' see that in https://github.com/Microsoft/msbuild/pull/1347/files